Automattic/jetpack
# CLI: make add-pr-num changelogger option work

#### Changes proposed in this Pull Request:

- Surface the `add-pr-num` option in the Jetpack CLI `changelog` & `release` commands.
- Update `changelogger-release.sh` with a `-p` switch to pass along the `--add-pr-num` option to changelogger.

#### Jetpack product discussion

Follow-up to https://github.com/Automattic/jetpack/pull/24045

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

- `tools/changelogger-release.sh -h` should show help for the `-p` switch to pass `--add-pr-num` to changeloggger.
- `jetpack changelog write --help` should show the `add-pr-num` option.
- `jetpack release --help` should show the `add-pr-num` option.

Example usage:

- Adding PR numbers with `jetpack changelog write plugins/starter-plugin --add-pr-num --use-version 1.0.0` should work.
- Adding PR numbers with `jetpack release plugins/jetpack changelog -a --add-pr-num` should work.
- Adding PR numbers with `jetpack release plugins/jetpack amend -b --add-pr-num` should work.

---

- [ ] Todo: update internal FG with info about add-pr-num option.
